.github/workflows/build.yml

```diff
@@ -41,4 +42,4 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test package source code for errors
       run: |
-        pytest
+        python -m pytest
```

Comment:
Running `pytest` should be fine

Reply:
Nope! This is the second major fix of this PR. Running `pytest` won't install `epispot` from the source because for some reason there is an installation issue there. Instead, we have to use the `epispot/` directory from the Python `Namespace` in `pytest`. In order to do this, we have to run `pytest` as a module.